tests/mocks/commit_select_buffer.lua

```diff
@@ -0,0 +1,18 @@
+local CommitSelectViewBuffer = require("neogit.buffers.commit_select_view")
+local git = require("neogit.lib.git")
+
+local M = {
+  values = {},
+}
+
+--- Add a rev name to the mocked list of commits the user selected.
+--- @param rev string the rev name of the commit which the user will select once `CommitSelectViewBuffer:open_async()` is called
```

Comment:
```suggestion
---Add a rev name to the mocked list of commits the user selected.
---@param rev string the rev name of the commit which the user will select once `CommitSelectViewBuffer:open_async()` is called
```

I don't know the reason, but lua-lsp doesn't want a space here 🤷🏼